hazelcast-client-new/src/main/java/com/hazelcast/client/spi/impl/ClientInvocationServiceSupport.java

```diff
@@ -96,10 +96,17 @@ protected void send(ClientInvocation invocation, ClientConnection connection) th
         ClientMessage clientMessage = invocation.getClientMessage();
         if (!isAllowedToSendRequest(connection, invocation) || !writeToConnection(connection, clientMessage)) {
             final int callId = clientMessage.getCorrelationId();
```

Comment:
No, they are not. I just checked operation call id it seems they are long, but as far as I remember client call ids were always int.